.github/workflows/links-fail-fast.yml

```diff
@@ -0,0 +1,17 @@
+name: Links (Fail Fast)
```

Comment:
What does "(Fail Fast)" mean in this context? Why isn't it just links?

Reply:
Renamed to links